site/sfguides/src/connectors_native_sdk_java_example/connectors_native_sdk_java_example.md

```diff
@@ -224,58 +224,11 @@ After all those steps are finished the connector will be ready to start ingestin
-Completing the prerequisites is not required, 
-but it is recommended to ensure smoother configuration process later.
-In the case of the example GitHub connector there are 3 things that need to be taken care of before going further:
-- preparing a GitHub account
-- creating a personal access token for that GitHub account
-- configuring external access integration inside Snowflake
-
-For the GitHub related prerequisites please check the [documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
-
-#### External Access
-Setting up external connectivity objects inside Snowflake requires the GitHub prerequisites to be completed. 
-External connectivity objects have to be created on the account level by the `accountadmin`.
-
-In order to access external resources the connector requires 3 objects to be configured:
-- [network rule](https://docs.snowflake.com/en/developer-guide/external-network-access/creating-using-external-network-access#creating-a-network-rule-to-represent-the-external-network-location)
-- [secrets](https://docs.snowflake.com/en/sql-reference/sql/create-secret)
-- [external access integration](https://docs.snowflake.com/en/developer-guide/external-network-access/creating-using-external-network-access)
-
-The following commands create necessary objects. Names can be customized and will be needed during connector Wizard phase later.
-
-```snowflake
-CREATE DATABASE GITHUB_SECRETS;
-
-CREATE OR REPLACE NETWORK RULE GH_RULE
-MODE = EGRESS
-TYPE = HOST_PORT
-VALUE_LIST=('api.github.com:443');
-
-CREATE OR REPLACE SECRET < GITHUB_TOKEN > TYPE=GENERIC_STRING SECRET_STRING='< PASTE API TOKEN >';
-
-CREATE OR REPLACE EXTERNAL ACCESS INTEGRATION GITHUB_INTEGRATION
-ALLOWED_NETWORK_RULES = (GH_RULE)
-ALLOWED_AUTHENTICATION_SECRETS = ('GITHUB_SECRETS.PUBLIC.< GITHUB_TOKEN >')
-ENABLED = TRUE;
-```
+Completing the prerequisites is not required, but it is recommended to ensure smoother configuration process later.
 
-#### Granting privileges to the application
-
-The application needs to have usage the following privileges on the created objects:
-- usage on the external access integration
-- usage on the database and the schema with the secrets
-- read on the secret itself
-To know the application instance name please check the `Makefile` or logs from the installation process.
-
-This translates to queries like those:
-```snowflake
-GRANT USAGE ON INTEGRATION GITHUB_INTEGRATION TO APPLICATION < application name >;
-
-GRANT USAGE ON DATABASE GITHUB_SECRETS TO APPLICATION < application name >;
-GRANT USAGE ON SCHEMA GITHUB_SECRETS.PUBLIC TO APPLICATION < application name >;
-GRANT READ ON SECRET GITHUB_SECRETS.PUBLIC.GITHUB_TOKEN TO APPLICATION < application name >;
-```
+In the case of the example GitHub connector there are 2 things that need to be taken care of before going further:
+- preparing a GitHub account
+- confirming access to ingested GitHub repositories
```

Comment:
```suggestion
- confirming access to the GitHub repositories for ingest
``` 
?

Reply:
I changed it to `confirming access to the GitHub repositories you want to ingest`